test/e2e/framework/test_context.go

```diff
@@ -335,14 +335,17 @@ func ViperizeFlags() {
 }
 
 func createKubeConfig(clientCfg *restclient.Config) *clientcmdapi.Config {
+	const (
+		tokenFile = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
No need for a full const block (or for this to be a const in general)

Reply:
Done. thanks @mikedanese